Makefile

```diff
@@ -85,8 +85,8 @@ docker-build: load-design-system-templates  ## Build Docker container
 	docker compose pull
 	docker compose build
 
-.PHONY: docker-start
-docker-start:  ## Start Docker containers
+.PHONY: docker-up
+docker-up:  ## Start Docker containers
```

Comment:
Is it necessary to remove docker-start as it's still part of README instructions on Using Docker - Build and Start the Container?

Reply:
Well spotted! It's not actually removed, I've only renamed the make command so that it's consistent with the Docker command it uses. I forgot to rename it in the README so will do that 👍

Reply:
Make sure to update the README to reflect this change as well! I have requested changes for this =)